addons/html_editor/static/tests/insert/paragraph_break.test.js

```diff
@@ -147,6 +151,7 @@ describe("Selection collapsed", () => {
                         highlightedValue: "ab\ncd",
                         textareaRange: 3, // "ab\n[]cd"
                     },
+                    contentAfter: wrapInPreWithData("ab\ncd", { html: "ab<br>cd" }) + "[]",
```

Comment:
Wait, the content of the PRE is duplicated in db then? Once really in the PRE itself and then a second time in a data attribute?